src/main/java/bit/Deadline.java

```diff
@@ -0,0 +1,47 @@
+package bit;
+
+import java.time.DateTimeException;
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+public class Deadline extends Task {
```

Comment:
Similarly, perhaps can include comment header for classes and public methods :)

Reply:
I will try and add them ASAP